ui/app/src/elements/components/table-filter-map.ts

```diff
@@ -35,6 +35,11 @@ export class DashboardFilterMap extends LitElement {
   @property()
   private _allAssessments;
 
+  @property()
+  private _allAssessmentsSubscriber = new StoreSubscriber(this, () =>
```

Comment:
This is my temp implementation to have a simple way for this to work. I think the original issue had something to do with the the logic around setting up subscriptions and unsubscribing, and how that relates to the components lifecycle - I wasn't able to make sense of all that logic at first glance so opted for this work around.

Reply:
There also seems to be a possible error with `flatFiltered()` method on the `DashboardFilterMap` component.